src/images/wavedrom/v-inst-table.adoc

```diff
@@ -0,0 +1,209 @@
+
+// [cols="4,1,1,1,8,4,1,1,8,4,1,1,8"]
```

Comment:
Column spacing and/or font size aren't optimized for the PDF output. It would be nice if columns didn't break into multiple-lines. In the templated used to generate the pdf in the v-spec repository they seem to be using smaller fonts through all the document and they don't have such a problem.

Reply:
PDF tables are a struggle tbh.  I've tried to fix them as best I can right now.  Moving to the new asciidoctor-web-pdf engine will fix them for good, but for now, asciidoctor-pdf won't render them nicely.